database/seeders/CategorySeeder.php

```diff
@@ -8,13 +8,26 @@
 
 class CategorySeeder extends Seeder
 {
-    /**
-     * Run the database seeds.
-     *
-     * @return void
-     */
-    public function run()
-    {
-        Category::factory()->create();
-    }
+	/**
+	 * Run the database seeds.
+	 *
+	 * @return void
+	 */
+	public function run()
+	{
+		Category::insert([
+			[
+				'title' => 'Numbers',
+				'description' => 'Ready to learn how to count from 1-100+ with Japanese numbers? Let\'s take a look at the Japanese numbers 1 - 10 in the Sino-Japanese Number system, which is most common.',
+				'created_at' => now(),
```

Comment:
@brixtuyor no need to put timestamps such `created_at' => now()` cause this is auto generated when you create. (unless you put it in fillable then you need to specify value)

Do the same for the rest of the models and kindly check if this is working.

Reply:
@roseaugusto Initially, I did not include the timestamps in the seeders. However, whenever I run the seeder files, the timestamps in the database are all null

Reply:
Let me try one more time...

Reply:
@roseaugusto I tried many times but the timestamps do not auto generate

Reply:
@brixtuyor okay. This should apply anyways if we use `create` but you are using `insert` so for now, just ignore this comment. Thanks!